### DISCUSSION TITLE
Request support for micropelt iTRV

### DISCUSSION THREAD

**Discussion by GerhardKreuzer** [mak-gitdev/HA_enoceanmqtt#125] [Request]
Hi,
there were very intersting devices out called MVA008 or MVA009, radiator valves harvesing energy from the difference of the radiator temperature and the room temperature and able to set the valve to a %age to control the radiator.
https://www.micropelt.com/en/products/enocean/mva008
They use the EnOcean euipment profile A5-20-01, maybe it isn't that hard to add this devices?
With best regards
Gerhard

**Comment by mak-gitdev** [comment]
Hi @GerhardKreuzer,

A5-20-01 is already available 🙂.
Please be aware of this issue #112, the next release will fix it.
